modules/percolator/src/main/java/org/elasticsearch/percolator/QueryAnalyzer.java

```diff
@@ -496,7 +504,7 @@ private static int minTermLength(Set<QueryExtraction> extractions) {
     }
 
     private static BytesRef smallestRange(Set<QueryExtraction> terms) {
-        BytesRef min = terms.iterator().next().range.interval;
+        BytesRef min = UnicodeUtil.BIG_TERM;
```

Comment:
It works for strings by using illegal characters. However here, ranges use a binary encoding and all values are allowed. So we should probably use `null` as a sentinel value.